js/__internal/grids/grid_core/column_fixing/m_column_fixing.ts

```diff
@@ -364,10 +366,10 @@ const baseFixedColumns = {
     }
   },
 
-  getColumns(rowIndex, $tableElement) {
```

Comment:
getColumns with second parameter was called only inside ColumnsView.setColumnWidths which was called from [here](https://github.com/DevExpress/DevExtreme/pull/25129/files#diff-e3426ecdc2d06abce3f678834b5e5ee20af76603e95ae2517fc3fedd850d69d7L453).

But now there is new method setFixedTableColumnWidths and this function never being called with second parameter now